packages/config/config/devices/0x0345/templates/swidget_template.json

```
@@ -0,0 +1,1504 @@
+{
+	"power_1_high_entry_value": {
+		// param 1
+		"label": "Power 1 High Entry Value",
+		"description": "Used to set the power value for Power 1 (the controlled load) that indicates entry into a Power 1 Low power state.",
```

Description is still very long - each of those will show up in UIs in full. Also, this description doesn't match the label of the param it seems. I suggest changing it to
```suggestion
		"label": "Power 1: High Entry Threshold",
```

On the other hand, you explain what "Power 1" is here, but I'm not sure I understand what "Power 2" is. Maybe rename "Power 1" to "Load" and do something similar for "Power 2"?
```suggestion
		"label": "Load: High Power Entry Threshold",
```

---

Anyways, if we do this, all related parameters should be renamed in a similar fashion.

---

I'm removing the descriptions as they are causing problems. I am leaving the labels in a format the manufacturer would prefer (that matches their zwave documentation)